### ISSUE TITLE
Brainstorming

### ISSUE THREAD

**Issue by gmmcal** [LandingJobs/404-hack#1]
Ideas for the page

**Comment by nuvan** [comment]
I was thinking in a person (or just a pair of hands) browsing the job ads in a old newspaper...something in those lines.

**Comment by joaobborges** [comment]
A crashed rocket in somekind of planet and maybe Elon Musk crying out loud.

**Comment by msssequeira** [comment]
@nuvan +1

**Comment by jayquest** [comment]
This page is not here anymore. "She" found a new job on Landing.jobs!

**Comment by tiagocesar** [comment]
I think it can offer a range of "easter eggs" - I particularly like this idea: http://www.aacd.com.br/404/

**Comment by tiagocesar** [comment]
Speaking now of Landing.Jobs... It should check if the user is logged in. If he/she is, then present a job position that fits him/her (if you have how to gather that). If it's a company, I don't know what to show

**Comment by lapa182** [comment]
Based on what @tiagocesar said, if possible to see if the user is logged in:

"Page not found, but we found something for you".

Based on Twitter 404, they had a wale to say that something was wrong, I still think that using a sardine lost in space is nice :P.

**Comment by douglasrbrito** [comment]
It's possible to merge the ideas from @nuvam and  @jayquest, to make an image from Landing Jobs looking for a new job on newspapers.

**Comment by lapa182** [comment]
@douglasrbrito I guess changing newspaper for an iPad or something like that would be better.

**Comment by gmmcal** [comment]
keep throwing ideas, no matter how crazy they seem to be. I think we will have an awesome project here!

**Comment by douglasrbrito** [comment]
@lapa182 It's valid too, but I think that @nuvan was trying to send the message from a offline page, and the newspaper would represent better this idea. Don't you think?

**Comment by nogdacosta** [comment]
so, based on the name "Landing.jobs".... an illustration of a Rocket missing the Landing pad (not exploding) and some awesome copy saying something like "Rocket science may be hard...but landing your next job it's easier..." or "OK, quick, look at this job openings and don't let the rocket explode"

**Comment by Wowca** [comment]
@douglasrbrito I don't agree with you on 404 being an offline page - it's "Not Found" by definition, so I guess it shouldn't be communicated to the end-users as offline.

Regarding @lapa182 and @nogdacosta proposals - I think they are awesome but as a user I expect that a 404 page just notifies me that the content is not found and guides me back to where I came from. I don't think it should be used to display some other dynamically generated content as I wasn't looking for that. 

I guess what I'm trying to say is - just refresh the current page to match your current awesome front-page design.

**Comment by douglasrbrito** [comment]
@wowca you got a point! Thinking on this way I like the idea of @lapa182

**Comment by nogdacosta** [comment]
@Wowca I believe it depends...If you were looking for a job opening in a certain company, and somehow that page no longer exists, independent of the reason, I think that we should get a message that would point us to the right page or to job openings similiar to that or other job opening in the same company.

Because we're talking about general 404, I would keep an awesome Illustration (space rocket, or the idea from @nuvan ) and a simple message that would take the user back home, or to the previous page (depending on the user flow inside the Landing.jobs site).

But if we can get creative, and decipher what the user is looking for, that would be great.

**Comment by nuvan** [comment]
@douglasrbrito @jayquest It was exacly what I was thinking. "No Landing here, try some dead trees". Nevertheless, I think that @lapa182's idea ( sardine in space ) makes a lot of sense since it makes use of a historic milestone for Landing.jobs. Maybe someone reading a newspaper with some sardines flying in makshift baloons in the air?

**Comment by lapa182** [comment]
With an easter egg, using konami code (or when the user hover the newspaper area) appearing and sardine saying TOASTY! (I'm not kidding).

http://www.myinstants.com/instant/toasty-d/

**Comment by Wowca** [comment]
@nogdacosta 

> But if we can get creative, and decipher what the user is looking for, that would be great.

Contextual error pages would actually be pretty awesome! It just depends on what data @gmmcal has regarding what the user is looking at and how much spaghetti code you would need to write to support this feature.

**Comment by pcbo** [comment]
What about background music? Everybody thinks about the visual (eye sight) part but we always forget other senses... example: https://soundcloud.com/pedroferraz/kamali-main-theme

**Comment by jpita** [comment]
Something like the 404 from github, with a moving background, for desktop would be the mouse movement and on mobile the gyro movement

**Comment by hmatinho** [comment]
Dudes just get an open source version of this 
https://m.youtube.com/watch?v=McAhSoAEbhM

**Comment by tunessofia** [comment]
or present a gif, from the ones that you guys are collecting by the short questionnaire, its simple and still funny

**Comment by tiagocesar** [comment]
The first thing to do is define what's the aim: To be funny? To be serious? To be functional? Any combination of the former?

A 404 page serves the purpose of putting the user in the right way, showing him how the "forest" is "mapped" - and providing him direct access to the information he wants to consume. Maybe something fun should be avoided, since traditional companies tend to complain silently about such things (I work using shorts and get the usual condemning looks on the elevator from the employees of the other companies in the same building - even if they also work with IT).

**Comment by gmmcal** [comment]
Hey guys, I'm getting the ideas and will add them to a trello board for upvoting next week. conversation is locked, for now.

**Comment by gmmcal** [comment]
Hi everyone. 404 page is on halt due to the upcoming holidays. On january we will continue it :D
